packages/block-library/src/avatar/hooks.js

```diff
@@ -76,14 +76,23 @@ export function useUserAvatar( { userId, postId, postType } ) {
 		},
 		[ postType, postId, userId ]
 	);
+	const defaultAvatar = useDefaultAvatar();
+	if ( authorDetails && ! authorDetails?.avatar_urls ) {
+		return {
+			src: defaultAvatar,
+			minSize: 24,
+			maxSize: 96,
```

Comment:
Thanks for the PR!

Here, maxSize will be different than the value we got from `getAvatarSizes( sizes )`

```
function getAvatarSizes( sizes ) {
	const minSize = sizes ? sizes[ 0 ] : 24;
	const maxSize = sizes ? sizes[ sizes.length - 1 ] : 96;
	const maxSizeBuffer = Math.floor( maxSize * 2.5 );
	return {
		minSize,
		maxSize: maxSizeBuffer,
	};
}
```
So if Avatar is disabled, it will appear with no errors, but with different size maximum value than if Avatar is enabled on discussion settings.